run.py

```diff
@@ -240,5 +254,22 @@ def grab_screen(n):
     return callback.get_screen_grab()
 
 
+@app.callback(
+    Output('calendar-output', 'children'),
+    [Input('Previous-change_mth', 'n_clicks'), Input('Next-change_mth', 'n_clicks')])
+def next_month(previous_clicks, next_clicks):
+    if previous_clicks > 0:
+        return change_previous_month(CalendarTimeObj)
+    if next_clicks > 0:
+        return change_next_month(CalendarTimeObj)
+
+
+@app.callback(
+    Output('live-calendar', 'children'),
+    [Input('calendar-interval-component', 'n_intervals')])
```

Comment:
In fact I'm not sure if generating whole calendar every 1 second is good. What things change during this one second?